File: exercicios_py/secao7/exercicio2.py
#Algoritmo que conte de 1 a 100, e a cada múltiplo de 10, emita uma mensagem:
#"Múltiplo de 10"

for i in range(1, 101):
    if i%10 == 0:
        print("Múltiplo de 10: ")
    print(i)

File: exercicios_java/exercicios_java/src/secao7/exercicio2.java
//Algoritmo que conte de 1 a 100, e a cada múltiplo de 10, emita uma mensagem:
//"Múltiplo de 10"

package secao7;

public class exercicio2 {
    public static void main(String[] args){
        for (int i=1;i <= 100; i++){
            System.out.println(i);
            if (i%10 == 0){
                System.out.println("Múltiplo de 10");
            }
        }
    }
}
